frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -199,6 +198,17 @@ if Device:hasKeys() then
                     G_reader_settings:flipNilOrTrue("enable_back_history")
                 end,
             },
+            {
+                text = _("Invert PageTurn Buttons"),
+                enabled_func = function() return Device:hasKeys() end,
+                checked_func = function()
+                    return G_reader_settings:isTrue("input_invert_buttons")
+                end,
+                callback = function()
+                    G_reader_settings:flipNilOrFalse("input_invert_buttons")
```

Comment:
`input_invert_buttons` could be made more explicit, like : `input_invert_page_turn_keys`